lang/semgrep-grammars/src/semgrep-julia/grammar.js

```diff
@@ -20,6 +20,28 @@ module.exports = grammar(base_grammar, {
   rules: {
     semgrep_ellipsis: $ => '...',
 
+    // Metavariables
+    identifier: ($, previous) => {
+      return token(
+        choice(
+          previous,
+          // We allow an identifier to be the same as an identifier, but prefixed
+          // with a dollar sign.
+          // We could use a more specific regular expression, which would match
+          // metavariables more faithfully, but either:
+          // 1) this would result in cases where we tokenize as an identifier or
+          // an interpolation expression where we don't want to, depending on how
+          // we set up the precedence.
+          // 2) this causes tokenization issues with interpolation expression that
+          // have prefixes which are metavariables, such as $Pack, which is parsed
+          // as two tokens -- an identifier "$P" and an identifier "ack".
+
+          // It's better to allow anything prefixed with a dollar sign to be an identifier,
+          // then sort it out in generic translation.
```

Comment:
Please add a comment also here about the fact that this leads to new parsing failure and this xfail hack.
Maybe you should show an example of interpolation expression and show how now
this is parsed differently.